ili/embedding/fcn.py

```diff
@@ -50,4 +50,11 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         Returns:
             torch.Tensor: data
         """
+        if not hasattr(self, "mlp"):
+            if len(x.shape) == 1:
+                x = x.view(-1, 1)
+            elif len(x.shape) > 2:
+                x = x.view(x.shape[0], -1)
```

Comment:
Please add both of these cases to the unit tests

Reply:
I realized the first one doesn't occur because our Loaders take care of the batch shaping for us

The second one will only occur in Lampe, where complex data structures are allowed (sbi only does vector regression). I added a test of this.